src/Apps/Artist/Components/ArtistHeader/ArtistHeaderImage.tsx

```diff
@@ -4,6 +4,9 @@ import { FC } from "react"
 import { maxDimensionsByArea, resized } from "Utils/resized"
 import { BREAKPOINTS, Media } from "Utils/Responsive"
 import { Link } from "react-head"
+import { getENV } from "Utils/getENV"
+
+const IS_GOOGLEBOT = getENV("IS_GOOGLEBOT")
```

Comment:
Is this 'safe' to do in a `const` like this versus deferred behind a `() =>` evaluation? (just wondering if this is loaded 'once')

Reply:
Yep should be safe!

code executes on the server _and_ again on the client. so in a server context it'll read from that weird global node cache thing, and on client from the window